bridges/evm/contracts/IBridge.sol

```diff
@@ -1,15 +1,36 @@
 pragma solidity 0.5.14;
 pragma experimental ABIEncoderV2;
 
+
 interface IBridge {
+    /// Request packet struct is similar packet on Bandchain using to re-calculate result hash.
+    struct RequestPacket {
+        string clientId;
+        uint64 oracleScriptId;
+        string params;
+        uint64 askCount;
+        uint64 minCount;
+    }
+
+    /// Response packet struct is similar packet on Bandchain using to re-calculate result hash.
+    struct ResponsePacket {
+        string clientId;
+        uint64 requestId;
+        uint64 ansCount;
+        uint64 requestTime;
+        uint64 resolveTime;
+        uint8 resolveStatus;
+        string data;
+    }
+
     /// Helper struct to help the function caller to decode oracle data.
     struct VerifyOracleDataResult {
```

Comment:
I don't think this is needed anymore? Why can't `relayAndVerify` returns `(RequestPacket memory, ResponsePacket memory)`?